eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/list/SynchronizedPutFastListMultimap.java

```diff
@@ -170,4 +170,9 @@ public void value(K key, MutableList<V> list)
     {
         return this.collectValues(function, FastListMultimap.<K, V2>newMultimap());
     }
+
+    public MutableListMultimap<K, V> asSynchronized()
+    {
+        return SynchronizedMutableListMultimap.of(FastListMultimap.newMultimap(this));
```

Comment:
I'm still not sure about this. Calling SynchronizedPutFastListMultimap.asSynchronized() is weird, and we need to think about it carefully. Usually you'd have one because you just called iterable.asParallel(...).groupBy(...). If you call asSynchronized() and it takes O(n) time, it defeats the purpose of whatever came earlier.  Maybe the best solution is to throw Unsupported. Not sure yet.